src/resource-archive/index.ts

```diff
@@ -83,7 +82,7 @@ class Watcher {
 
     // The second promise wraps a network idle timeout. This uses playwright's built-in functionality to detect when the network
     // is idle.
-    const networkIdlePromise = this.page.waitForLoadState('networkidle').finally(() => {
+    const networkIdlePromise = page.waitForLoadState('networkidle').finally(() => {
```

Comment:
I was unable to remove this Playwright dependency (`page`) by directly passing the `waitForLoadState` function instead of `page`. Cypress CDP won't need to use this `idle()` method at all; if that's the case, I'll extract `idle()` from here and put it into `playwright-api/createResourceArchive` in a future PR.